Fix:
memcacheq.c:2222:15: error: implicit declaration of function 'daemonize' [-Werror,-Wimplicit-function-declaration]
https://github.com/stvchu/memcacheq/issues/7
--- memcacheq.c.orig	2014-12-16 22:29:10.000000000 -0600
+++ memcacheq.c	2021-07-16 02:40:55.000000000 -0500
@@ -69,6 +69,7 @@
 /*
  * forward declarations
  */
+static int daemonize(int nochdir, int noclose);
 static void drive_machine(conn *c);
 static int new_socket(struct addrinfo *ai);
 static int server_socket(const int port, const bool is_udp);
